frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -348,6 +353,8 @@ public void run() {
     @UIEffect
     public void popupMenuWillBecomeInvisible(PopupMenuEvent popupMenuEvent) {
       graphTable.setRowSelectionAllowed(false);
+      // So that the selection (and thus the area where tooltip is shown) has again the possible smallest size
```

Comment:
Nit (didn't notice previously):
```suggestion
      // So that the selection (and thus the area where tooltip is shown) has again the smallest possible size
```